dashboard/config/scripts_json/allthethings.script_json

```diff
@@ -4,8 +4,10 @@
     "wrapup_video_id": null,
     "login_required": false,
     "properties": {
-      "lesson_extras_available": true,
-      "hideable_lessons": true
+      "hideable_stages": true,
+      "stage_extras_available": true,
```

Comment:
nit: I know this this diff is noisy but can you remove `hideable_stages` and `stage_extras_available` here? I'm not sure how they wound up back in this file but they're deprecated

Reply:
Yes definitely! I was having a hard time figuring out the merge, and anything that was put in here happened automatically- thank you for the help!